source/Nevermore/Advanced/SelectBuilders/TableSelectBuilder.cs

```diff
@@ -35,5 +37,17 @@ public override ISelectBuilder Clone()
         {
             return new TableSelectBuilder(From, IdColumn, new List<IWhereClause>(WhereClauses), new List<GroupByField>(GroupByClauses), new List<OrderByField>(OrderByClauses), ColumnSelection, RowSelection);
         }
+
+        public override void AddWhere(UnaryWhereParameter whereParams)
+        {
+            if (From.ColumnNames.Contains(whereParams.FieldName))
```

Comment:
@adam-mccoy just a heads up on this one; `SelectAllColumnsTableResolver` will return the columns as `new [] { "*" }`.

It's not wired in by default since the most recent release (we now use `JsonLastTableColumnNameResolver` which _will_ return the column names) but this logic might need to be flipped to maintain backwards compat with the old resolver.